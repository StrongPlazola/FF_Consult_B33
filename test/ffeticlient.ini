##########################################
# FlexFlow ETI Client
# Configuration file.
##########################################
#Server=10.106.230.171
#Port=8063
Server=10.106.237.144
Port=8091
User=ffeti
Password=feti
#Station=PHI-TLA-ASSY-BASE-LID-FRONT-L11-GLP
#Station=PHI-AUT-PCM-TIM
#Station=PHI-PRESSFIT-COMPLETE
Station=PHI-PRESSFIT