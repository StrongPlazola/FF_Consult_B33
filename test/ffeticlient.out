{ERROR} GLBFLG254000411 3020- PHI-PRESSFIT Pass Goto  PHI-PRESSFIT-INSPECTION-(3020)- Ready for Station Type: : The FID is in Unit State: 3020- PHI-PRESSFIT Pass Goto  PHI-PRESSFIT-INSPECTION-(3020)- Ready for Station Type: 
